src/lib/vault/getFormSubmission.ts

```diff
@@ -0,0 +1,40 @@
+import { GetCommand } from "@aws-sdk/lib-dynamodb";
+import { dynamodbClient } from "@lib/awsServicesConnector";
+
+interface FormSubmission {
+  content: string;
+}
+
+export async function getFormSubmission(
+  formId: string,
+  submissionName: string,
+): Promise<FormSubmission | undefined> {
+  try {
+    const response = await dynamodbClient.send(
+      new GetCommand({
+        // biome-ignore lint/style/useNamingConvention: <explanation>
+        TableName: "Vault",
+        // biome-ignore lint/style/useNamingConvention: <explanation>
+        Key: { FormID: formId, NAME_OR_CONF: `NAME#${submissionName}` },
+        // biome-ignore lint/style/useNamingConvention: <explanation>
+        ProjectionExpression: "FormSubmission",
+      }),
+    );
+
+    if (response.Item === undefined) {
+      return undefined;
+    }
+
+    return {
+      content: response.Item.FormSubmission,
+    };
+  } catch (error) {
+    console.error(
+      `[dynamodb] Failed to get form submission. FormId: ${formId} / SubmissionName: ${submissionName}. Reason: ${JSON.stringify(
```

Comment:
We will probably be looking at integrating Pino or something similar later on to replace our existing console logs.

In the GC Forms web app we are not always able to know what happened in the code because our logs are sometimes missing some context (layer, inputs, outputs, ect..). 

Here is an idea I would like to propose for a future log function:

````
log({
  level: "error",
  message: "Failed to retrieve form submission",
  cause: "Throughput exceeds the current capacity of your table or index",
  context: {
    layer: "dynamodb",
    inputs: {
      variable1: 2,
      variable2: "abc123",
    },
  },
});
````